_guides/004-platforms.md

```diff
@@ -47,6 +47,24 @@ sections:
             title: Production deployment tips
             description: Production deployments
             module: platforms/kubernetes/howto/production
+      -
+        title: Use cases
+        description: Examples for operations
+        module: platforms/kubernetes/examples
+        subitems:
+          -
+            title: MySQL
+            description: Create a MySQL stateful set
+            module: platforms/kubernetes/examples/mysql
+          -
+            title: Monitor volumes
```

Comment:
This is from the copy and paste from the subitems of howto. Both, monitor and production tips shouldn't be in the use cases. 

If we would go and merge this, we would break the menu in the docs. 

(By removing them and creating a new commit is enough).

Reply:
Fixed.